packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,73 +4,51 @@ This log was last generated on Tue, 23 Apr 2024 17:54:31 GMT and should not be m
-- Add UI test for ACS captions  ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Move survey and captions to concept page ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Reaction animation enforcing of the frame API ([commit](https://github.com/azure/communication-ui-library/commit/not available) by mbellah@microsoft.com)
-- Add endCallButton option ([commit](https://github.com/azure/communication-ui-library/commit/not available) by jiangnanhello@live.com)
-- Captions Kind Changed event and store captions Kind inside context ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Improved closed captions storybook based on feedback ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Dismiss drawer on reaction click on mobile ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 2684369+JamesBurnside@users.noreply.github.com)
+
+### Rich Text Editor - Public Preview
+
+The Web UI RichTextSendBox and MessageThread components now support rich text formatting to generate HTML content for chat messages. This feature includes support for: Different text styles (bold, italic and underline)Bulleted and numbered listsChanging indentation for textAdding and updating tables
+
+### Closed Captions - Public Preview
+
+We are excited to announce that Azure Communications Services has recently added the Closed Captions feature. With closed captions, developers can provide a textual representation of the audio content in their videos, making it easier for users who are deaf or hard of hearing to follow along. Developers can enable this functionality today through our composites (e.g CallComposite, CallWithChatComposite). Azure Communications Services Closed Captions supports: Enable and disable closed captions in the call, Change spoken language, Being able to use captions outside of teams scenario
+
 ### Improvements
+
 - Renamed "file" to "attachment" for File Sharing Upload Flow ([PR #4369](https://github.com/azure/communication-ui-library/pull/4369) by 109105353+jpeng-ms@users.noreply.github.com)
```

Comment:
Every changelog item should be started from noun or V1 (verb in the base form) to be consistent with the previous releases. It looks like mostly V2 are used in this release changelog

Reply:
Will update to the correct grammar